feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/usecases/response/CreateIdentifyResponseUseCase.kt

```diff
@@ -22,154 +21,79 @@ internal class CreateIdentifyResponseUseCase @Inject constructor(
         results: List<Serializable>,
     ): AppResponse {
         val isMultiFactorIdEnabled = projectConfiguration.multifactorId?.allowedExternalCredentials?.isNotEmpty() ?: false
-        val credentialFaceMatchResults = credentialResultsMapper(results, projectConfiguration, isFace = true)
-        val credentialFingerprintMatchResults = credentialResultsMapper(results, projectConfiguration, isFace = false)
 
         val currentSessionId = eventRepository.getCurrentSessionScope().id
-
-        val faceResults = credentialFaceMatchResults + getFaceMatchResults(results, projectConfiguration)
-        val bestFaceConfidence = faceResults.firstOrNull()?.confidenceScore ?: 0
-
-        val fingerprintResults = credentialFingerprintMatchResults + getFingerprintResults(results, projectConfiguration)
-        val bestFingerprintConfidence = fingerprintResults.firstOrNull()?.confidenceScore ?: 0
-
         return AppIdentifyResponse(
             sessionId = currentSessionId,
             isMultiFactorIdEnabled = isMultiFactorIdEnabled,
             // Return the results with the highest confidence score
-            identifications = if (bestFingerprintConfidence > bestFaceConfidence) {
-                fingerprintResults.distinctBy(AppMatchResult::guid)
-            } else {
-                faceResults.distinctBy(AppMatchResult::guid)
-            },
+            identifications = getResults(results, projectConfiguration),
         )
     }
 
-    private fun getFingerprintResults(
+    /**
+     * Combines all of the matching results per SDK and returns up to [maxNbOfReturnedCandidates] results from the SDK with
+     * the highest overall score in descending order. Credential matches take precedence over direct matches.
+     *
+     * If there are any matches of [AppMatchConfidence.HIGH], only those will be returned,
```

Comment:
of = above?

Reply:
`AppMatchConfidence` is the enum of confidence bands, there is no band above HIGH.